app/migrations.go

```diff
@@ -20,6 +20,7 @@ const ContentExtractionConfigDefaultTrueMigrationKey = "ContentExtractionConfigD
 const PlaybookRolesCreationMigrationKey = "PlaybookRolesCreationMigrationComplete"
 const FirstAdminSetupCompleteKey = model.SystemFirstAdminSetupComplete
 const remainingSchemaMigrationsKey = "RemainingSchemaMigrations"
+const PostPriorityConfigDefaultTrueMigrationKey = "PostPriorityConfigDefaultTrueMigrationComplete"
```

Comment:
No need to export this variable. Or if exporting then better use it, instead of the hard-coded value in `testlib > store.go`.